src/Microsoft.Azure.SignalR/EndpointRouters/IEndpointRouter.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Azure.SignalR
+{
+    public interface IEndpointRouter : IMessageRouter
```

Comment:
what about `INegotiateRouter` and `IMessageRouter`?

Reply:
Tend to not make that big change